rsmtool/utils.py

```diff
@@ -856,12 +856,86 @@ def show_thumbnail(path_to_image, image_id):
         The id of the <img> tag in the HTML. This must
         be unique for each <img> tag.
 
-    Returns
-    -------
+    Displays
+    --------
     display : IPython.core.display.HTML
         The HTML display of the thumbnail image.
     """
-    return display(HTML(get_thumbnail_as_html(path_to_image, image_id)))
+    display(HTML(get_thumbnail_as_html(path_to_image, image_id)))
+
+
+def get_files_as_html(output_dir, experiment_id, file_format, replace_dict={}):
+    """
+    Generate HTML list items for each file name,
+    given output directory. Optionally pass a
+    replacement dictionary to use more descriptive
+    titles for the file names.
+
+    Parameters
+    ----------
+    output_dir : str
+        The output directory.
+    experiment_id : str
+        The experiment ID.
+    file_format : str
+        The format of the output files.
+    replace_dict : dict, optional
+        A dictionary which makes file names to descriptions.
+        Defaults to empty dictionary.
+
+    Yields
+    ------
+    html_string : str
+        HTML list item with file description and link.
+    """
+    html_string = ("""<li><b>{}</b>: <a href="{}" download>{}</a></li>""")
+    files = [f for f in os.listdir(output_dir) if f.endswith(file_format)]
+
+    for file in sorted(files):
+        relative_file = os.path.relpath(os.path.join(output_dir, file))
```

Comment:
Can we use `pathlib` throughput this function? And also, we should probably look up just the prefix in the replacement dictionary rather than the whole filename? So, just `betas` instead of `lr_betas.csv`? And we could make the code a bit more readable:

```python
files = Path(output_dir).glob('*.{}'.format(file_format))

for file in sorted(files):
    relative_file = file.relative_to(os.getcwd())
    relative_name = relative_file.stem.replace(experiment_id, '')

    if relative_name in replace_dict:
        descriptive_name = replace_dict[relative_name]
    else:
        descriptive_name_components = relative_name.split('_') 
        descriptive_name = ' '.join(descriptive_name_components).strip().title()

   yield ...
```